charts/elastic/1.3.0/values.yaml

```
@@ -329,6 +329,36 @@ logstash:
     tolerations: []
     pipelines: []
 
+enterprise_search:
```

```suggestion
enterpriseSearch:
```
Please change it in all values